src/notify/notify.c

```diff
@@ -231,7 +234,7 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case ARG_BOOTED:
-                        arg_booted = true;
+                        arg_action = ACTION_BOOTED;
```

Comment:
`parse_argv()` still uses `arg_booted`.
The commit `notify: introduce arg_action enum` cannot be built.